staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook_test.go

```diff
@@ -33,19 +34,22 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/runtime/serializer/json"
 	auditinternal "k8s.io/apiserver/pkg/apis/audit"
-	auditv1alpha1 "k8s.io/apiserver/pkg/apis/audit/v1alpha1"
+	auditv1beta1 "k8s.io/apiserver/pkg/apis/audit/v1beta1"
```

Comment:
Let's move beta tests to a separate file in the same way it's done for alpha tests. We'll have to do this in the future anyway